lib/gollum-lib/markup.rb

```diff
@@ -126,7 +126,7 @@ def render_default(data, format=:markdown, name='render_default.md')
       @format = format
       @name   = name
 
-      chain = [:YAML, :PlainText, :Emoji, :TOC, :RemoteCode, :Code, :Sanitize, :PlantUML, :Tags, :Render]
+      chain = [:YAML, :PlainText, :Emoji, :TOC, :RemoteCode, :Code, :Sanitize, :PlantUML, :Tags, :Render].reject {|filter| skip_filter?(filter)}
```

Comment:
You mean why in the `render_default` method? It's used in the tests at least, so filter skipping should work there too, I thought.